cli/template/extras/config/prettier.config.ts

```diff
@@ -0,0 +1,5 @@
+import { type Config } from "prettier";
+
+export default {
+  plugins: [require.resolve("prettier-plugin-tailwindcss")],
+} satisfies Config;
```

Comment:
Oh shit didnt know this was possible. Lets gooo

Reply:
😎

Reply:
ive removed the `require.resolve` afaik that was only useful if the config was not at the root of your monorepo? correct me if im wrong.
it seems to work

Reply:
nvm something isnt working, with or without require. i hope the plugin works with this update